### PR TITLE
completed RM

### PR DESCRIPTION
**Pull request by Fishgif**
Completed RM file.

Please check spelling

### CODE REVIEW

**Comment by Samw1s3** [comment]
Hey Marc, looks good , just on line 12 which is spelt wrong. If you could edit and the push again :D

File: README.md [hunk 1]
```diff
@@ -1,10 +1,38 @@
...
-SO I CAN reduce my chances of having awkward silences in my conversations
 
+I WANT TO be able to acces an app that will generate different conversational points.
+SO I CAN reduce my chances of having awkward silences in the conversation with my date.
 
 Acceptance Criteria
+
+1. Love Calculator
+WHEN I insert two names and submit
+THEN the LOVE CALCULATOR fetches the percentage match whuch is displayed on screen.
```
Comment (Samw1s3) [inline README.md:12]: Which is spelt wrong

File: README.md [hunk 2]
```diff
@@ -1,10 +1,38 @@
...
+THEN A rondom joke is generated
+
+4. Excuse Generator
+WHEN I click on the button of the excuse generator 
+THEN a random excuse is generated
+
+In this project, we have worked to create a polished UI.
+At least two server API's must be used.
+Client - side site storage must be used
+CSS famework must be used other than Bootstrap
```
Comment (Samw1s3) [inline README.md:29]: spelling - framework

File: README.md [hunk 3]
```diff
@@ -1,10 +1,38 @@
...
+
+2. Weather App
+WHEN I click on the WEATHER APP 
+THEN The weather for my location is displayed using an geolocator
+
+3. Joke Generator
+WHEN  I click on the button
+THEN A rondom joke is generated
+
+4. Excuse Generator
```
Comment (Fishgif) [inline README.md:22]: Could be something else

File: README.md [hunk 4]
```diff
@@ -1,10 +1,38 @@
...
+WHEN I click on the button of the excuse generator 
+THEN a random excuse is generated
+
+In this project, we have worked to create a polished UI.
```
Comment (Samw1s3) [inline README.md:26]: Add a title to this section?

**Review by Samw1s3 — APPROVED** [review]
good job...A+

**Pull request merged**